src/components/ScmRepositoryLink/ScmRepositoryLink.tsx

```diff
@@ -17,27 +19,46 @@ interface IScmRepositoryLinkProps {
  * @param url - the internal/external url for the SCM Repository
  * @param showClipboardCopy - whether to display the url as clipboard copy
  * @param showGerritButton - whether to display the Gerrit button
+ * @param isInline - whether to use inline style with external link action
  */
-export const ScmRepositoryLink = ({ url, showClipboardCopy = true, showGerritButton }: IScmRepositoryLinkProps) => {
-  const gerritButton = showGerritButton && (
-    <Button
-      component="a"
-      href={parseInternalRepositoryUrl({ internalUrl: url })}
-      target="_blank"
-      rel="noopener noreferrer"
-      variant="tertiary"
-      icon={<ExternalLinkAltIcon />}
-    >
-      Gerrit
-    </Button>
+export const ScmRepositoryLink = ({ url, showClipboardCopy = true, showGerritButton, isInline }: IScmRepositoryLinkProps) => {
+  const GerritButton = () => (
```

Comment:
Move this component above ScmRepositoryLink component.

Reply:
+1